src/RollImage.cpp

```diff
@@ -564,7 +576,7 @@ void RollImage::assignMidiKeyNumbersToHoles(void) {
 		return;
 	}
 
-	int shifting = rewindholemidi - midiKey[bestFitIndex];
+	int shifting = midiKey[bestFitIndex] - rewindholemidi;
```

Comment:
Before I go ahead and merge this otherwise straightforward PR, would you mind elaborating on this change, for my benefit?  The commit message reads "Refine candidate rewind hole detection; ...", but it's not otherwise mentioned.  Thanks!